### PR TITLE
Mention Non BM Agents as a TP feature

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.17+

Issue: No Jira

Link to docs preview: https://83996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/index.html

QE review: 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/hosted-control-planes-overview.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@
 {hcp-capital} is available by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#cluster_mce_overview[{mce} version 2.0 or later] on the following platforms:
 
 * Bare metal by using the Agent provider
-* Non-bare metal Agent machines
+* Non-bare-metal Agent machines, as a Technology Preview feature
```
Comment (xenolinux) [inline modules/hosted-control-planes-overview.adoc:14]: Per ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words
and SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_b

Changing `Non-bare metal` to `Non-bare-metal`

**Review by snarayan-redhat — COMMENTED** [review]
LGTM!

**Pull request merged**